### PR TITLE
Update gradle files

### PR DESCRIPTION
**Pull request by gunhansancar**
-Removed deprecated `com.github.dcendents:android-maven-gradle-plugin` plugin
-Updated publishing to the central location into `bintray.gradle`
-Updated all `build.gradle` files in each project

### CODE REVIEW

File: bintray.gradle [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        MyPublication(MavenPublication) {
+            pom.withXml {
+                asNode().with {
+                    getAt('packaging')[0].replaceNode { packaging('aar') }
+                    appendNode('name', this.publishDescription)
+                    appendNode('url', siteUrl)
+
+                    appendNode('licenses').with {
+                        appendNode('license').with {
+                            appendNode('name', 'The Apache Software License, Version 2.0')
```
Comment (devgabrielcoman) [inline bintray.gradle:33]: is this the actual license?

File: bintray.gradle [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                            appendNode('url', 'http://www.apache.org/licenses/LICENSE-2.0.txt')
+                        }
+                    }
+
+                    appendNode('developers').with {
```
Comment (devgabrielcoman) [inline bintray.gradle:38]: if we're doing this, does my name / account have to be here? thinking maybe we transition to an official SA one?

File: bintray.gradle (hunk 1 above)
Comment (gunhansancar) [reply bintray.gradle:33]: It is what we had so far. Didn't change it. If needed we can update it to something else.

File: bintray.gradle (hunk 2 above)
Comment (gunhansancar) [reply bintray.gradle:38]: AFAIK developers tag is needed if we publish to the sonatype maven central. But I couldn't find our repo over there which means we don't sync.

It is better to sync to Jcenter (which is superset of maven central) which doesn't have any requirements like developers etc. 

And then we can even remove `maven { url "http://dl.bintray.com/gabrielcoman/maven" }` personal maven repository links.

**Pull request merged**